custom_components/tuya_local/devices/windmill_airconditioner.yaml

```diff
@@ -0,0 +1,55 @@
+name: Windmill Air Conditioner
+products:
+  - id: ebd7ef1c98deabe827aukn
```

Comment:
THis appears to be in the format used for device id, rather than product id.

Reply:
Could you point me to where I could find the product ID? I found this "model ID" in the tuya iot "Query Things Data	Model" response `00000049xm` but I'm not sure if that's right either.

Reply:
No, that it also not right (I'm not sure what that is for, and I think it is only available from the cloud API).

The product id is listed in the API calls that return the local key, and is available during device discovery locally, though we currently only use the device discovery via tinytuya library to find IP addresses.

Reply:
@make-all found it. Anything else needed to get this merged?